core/input/input.cpp

```diff
@@ -821,7 +821,7 @@ void Input::joy_button(int p_device, int p_button, bool p_pressed) {
 
 	_THREAD_SAFE_METHOD_;
 	Joypad &joy = joy_names[p_device];
-	//printf("got button %i, mapping is %i\n", p_button, joy.mapping);
+	// printf("got button %i, mapping is %i\n", p_button, joy.mapping);
```

Comment:
This is wrong. Commented out code should not start with a space.

While we don't have a complete C++ style guide yet, most of what is in the [GDScript style guide](https://docs.godotengine.org/en/latest/getting_started/scripting/gdscript/gdscript_styleguide.html) that can apply to C++ does apply to the C++ code.

Reply:
I used a simple regex to replace the code, I'll change that after #35998 is reworked.